### PR TITLE
EL-3102 - Updating seed project to Angular 6

### PR DESCRIPTION
**Pull request by ashley-hunter**
- Updated the seed project to use Angular 6
- Made some improvements to our use of Type Definitions
- Added the globals patch so the current release works fine. This can be removed as part of the next release as we will also need to update the package.json in the seed project this release as its a minor version bump of UX Aspects.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,6 +60,6 @@
     "protractor": "~5.1.2",
     "ts-node": "~3.2.0",
     "tslint": "~5.9.1",
-    "typescript": "^2.5.3"
+    "typescript": "^2.7.2"
```
Comment (donna-marijne) [inline package.json]: I think we need to remove the caret here to ensure that it matches the CLI requirements:
````
ERROR in The Angular Compiler requires TypeScript >=2.7.2 and <2.8.0 but 2.9.2 was found instead.
````

**Review by donna-marijne — CHANGES_REQUESTED** [review]
Just one issue with a dependency, otherwise all working fine.

**Comment by ashley-hunter** [comment]
Thanks, should now be fixed

**Review by donna-marijne — APPROVED** [review]
All good!

**Pull request merged**